syntax = "proto3";

package test_me;

message State_Data {
  repeated float myVector = 1;
  }